### ISSUE TITLE
getSongsById

### ISSUE THREAD

**Issue by ghost** [sc4v3ng3r/flutter_audio_query#16]
crash application if memberIds contain only one song title.

**Comment by sc4v3ng3r** [comment]
membersIds property must not be the song titles. It must be the song ID(s) value(s).

**Comment by ghost** [comment]
```
getSongsFromIds(List<String> memberIds) async {
    print('memberIds:${memberIds.length}');
    if (memberIds.length == 0) {
      _songsFromIds = [];
    } else {
      print("here");
      print(memberIds.toString());
      _songsFromIds = await audioQuery.getSongsById(ids: memberIds);
      print(_songsFromIds.toString());
      notifyListeners();
    }
  } 
```

![Capture](https://user-images.githubusercontent.com/60932394/76314516-c6815e80-62f8-11ea-9815-60fc12399cf7.PNG)

**Comment by ghost** [comment]
if there are more than one memberId than it work fine. But there is only one memberId app crashes.

**Comment by mvarendorff** [comment]
Yup, I am facing the same issue. 

```dart
await FlutterAudioQuery().getSongsById(ids: ["0"]);
```

is all it takes to crash it.
```
I/System.out(32097): SongLoader::basicLoad java.lang.IllegalArgumentException: Cannot bind argument at index 1 because the index is out of range.  The statement has 0 parameters.
```
is probably the best attempt at an error message I got.

**Comment by mvarendorff** [comment]
@ahmedaslam11 what appears to be a usable workaround is to add an empty string in case the list of ids has length 1. It's not ideal but works until this might be resolved in the future.

**Comment by sc4v3ng3r** [comment]
fixed and closed #37
